src/main/java/Duke.java

```diff
@@ -1,10 +1,196 @@
+            executeListTasks();
+            break;
+        case COMMAND_COMPLETED_WORD:
+            executeCompleteTask(commandArgs);
+            break;
+        case COMMAND_TODO_WORD:
+            executeTodoTask(commandArgs);
+            break;
+        case COMMAND_DEADLINE_WORD:
+            executeDeadlineTask(commandArgs);
+            break;
+        case COMMAND_EVENT_WORD:
+            executeEventTask(commandArgs);
+            break;
+        case COMMAND_EXIT_WORD:
+            executeExitProgram();
+            break;
+//        default:
+//            executeTodoTask(commandType);
+//            break;
+        }
+    }
+
+    private static void executeExitProgram() {
+        showToUser(DASHES,MESSAGE_GOODBYE,DASHES);
+        System.exit(0);
+    }
+
+    private static void executeEventTask(String rawInput) {
+        String[] decodedInput = decodeInput(rawInput);
+        String description = getDescription(decodedInput);
+        String at = getAdditionalInfo(decodedInput);
+        createEventTask(description, at);
+    }
+
+    private static void createEventTask(String description, String at) {
+        taskList[listCount] = new Event(description, at);
+        showSuccessfulAdd();
+        listCount++;
+    }
+
+    private static void showSuccessfulAdd() {
+        System.out.println("Got it! I've added this task: ");
+        System.out.println(" " + taskList[listCount]);
+        int totalListCount = listCount+1;
+        System.out.println("Now you have " + totalListCount + " tasks in the list.");
+    }
+
+    private static String[] decodeInput(String rawInput) {
+        String[] decoded = new String[TASK_DATA_COUNT];
+        String[] splitByForwardSlash = rawInput.split("/",2);
+        decoded[TASK_DATA_INDEX_DESCRIPTION] = splitByForwardSlash[0];
+        String[] splitBySpace = splitByForwardSlash[1].split(" ",2);
+        decoded[TASK_DATA_INDEX_ADDITIONAL_INFO] = splitBySpace[1];
+        return decoded;
+    }
+
+
+    private static String getAdditionalInfo(String[] decodedInput) {
+        return decodedInput[1];
+    }
+
+    private static String getDescription(String[] decodedInput) {
+        return decodedInput[0];
+    }
+
+    private static void executeDeadlineTask(String rawInput) {
+        String[] decodedInput = decodeInput(rawInput);
+        String description = getDescription(decodedInput);
+        String by = getAdditionalInfo(decodedInput);
+        createDeadlineTask(description, by);
+    }
+
+    private static void createDeadlineTask(String description, String by) {
+        taskList[listCount] = new Deadline(description, by);
+        showSuccessfulAdd();
+        listCount++;
+    }
+
+    private static void executeTodoTask(String todoInput) {
+        createTodoTask(todoInput);
+    }
+
+    private static void createTodoTask(String todoInput) {
+        taskList[listCount] = new Todo(todoInput);
+        showSuccessfulAdd();
+        listCount++;
+    }
```

Comment:
A lot of similar methods are written specifically for each subclass of task. Perhaps think about how u can take advantage of polymorphism and abstraction to declare these methods in each of the subclasses instead.